src/inputs/input.stories.js

```diff
@@ -0,0 +1,33 @@
+import { storiesOf } from "@storybook/html";
+import { htmlStory } from "../story-helpers";
+import ejs from "ejs";
+import html from "./input.stories.html";
+import { radios, boolean, select } from "@storybook/addon-knobs";
+import "./input.css";
+
+storiesOf("OpenMRS Styleguide", module).add("Inputs", () => {
+  const inputType = select(
+    "Type",
+    {
+      Underlined: "underlined",
+      Filled: "filled",
+      Outlined: "outlined"
+    },
+    "outlined"
+  );
+
+  const isDanger = boolean("is Danger", false);
+
+  const determineStyle = (inputType, isDanger) => {
+    let styles = `omrs-input-${inputType}`;
+    isDanger ? (styles = `${styles} omrs-input-danger`) : "";
```

Comment:
It is poor practice to modify a variable inside of a ternary -- ternaries are meant for returning a new value.

If you want to modify a value instead returning a new value, use `if` `else`. When transitioning to React, it's common for people to think that `if` statements are somehow no longer okay. But that is not true.